django/thunderstore/repository/models/cache.py

```diff
@@ -111,3 +117,166 @@ def drop_stale_cache(cls):
                 entry.delete()
         for entry in cls.objects.filter(community=None).iterator():
             entry.delete()
+
+
+# TODO: unit tests
+class APIV1ChunkedPackageCache(SafeDeleteMixin):
+    community: Community = models.ForeignKey(
+        "community.Community",
+        related_name="chunked_package_list_cache",
+        on_delete=models.CASCADE,
+    )
+    index: DataBlob = models.OneToOneField(
```

Comment:
TODO: this can't be a OneToOneField since the same blob can be index in multiple caches if the contents didn't change.